javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,10 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var newArray = [];
+  var i = words.length;
+  while (i > 0) {
+    newArray.unshift(words[i - 1] + suffix);
+    i--;
+  }
```

Comment:
Is this just for practice with `while` loops? If so, I would stick to keeping this a positive movement through the array, going backwards through the array adds an unnecessary level of complexity to the code.